spyder/config/main.py

```diff
@@ -73,7 +73,11 @@
               'single_instance': True,
               'open_files_port': OPEN_FILES_PORT,
               'tear_off_menus': False,
+              'normal_screen_resolution': True,
               'high_dpi_scaling': False,
+              'high_dpi_custom_scale_factor': False,
+              'high_dpi_custom_scale_factors': '2;',

```

Comment:
Yeah, but that's something for users to try. The point is showing them that setting float factors is a possibility (and users on issue #4316 already made use of that possibility).